.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: php
 
-sudo: true
```

Comment:
don't we need `sudo: false`? (OK - looked at docs, you only need it if the repo was enabled before 20:15 - could still be nice to be explicit... maybe?)